web/auth/auth_test.go

```diff
@@ -34,40 +29,36 @@ import (
 	"github.com/cozy/cozy-stack/web/auth"
 	"github.com/cozy/cozy-stack/web/errors"
 	"github.com/cozy/cozy-stack/web/middlewares"
-	jwt "github.com/golang-jwt/jwt/v4"
+	"github.com/gavv/httpexpect/v2"
+	"github.com/golang-jwt/jwt/v4"
 	"github.com/labstack/echo/v4"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 const domain = "cozy.example.net"
 
-var JWTSecret = []byte("foobar")
-
-var ts *httptest.Server
-var testInstance *instance.Instance
-
-var jar http.CookieJar
-var client *http.Client
-var clientID string
-var clientSecret string
-var registrationToken string
-var sharingClientID string
-var altClientID string
-var altRegistrationToken string
-var csrfToken string
-var code string
-var refreshToken string
-var linkedClientID string
-var linkedClientSecret string
-var linkedCode string
-var confirmCode string
-
 func TestAuth(t *testing.T) {
 	if testing.Short() {
 		t.Skip("an instance is required for this test: test skipped due to the use of --short flag")
 	}
 
+	var JWTSecret = []byte("foobar")
+
+	var sessionID string
+
+	var clientID string
+	var clientSecret string
+	var registrationToken string
+	var altClientID string
+	var altRegistrationToken string
+	var csrfToken string
+	var code string
+	var refreshToken string
+
+	// TODO: Remove this print use to reomve an error for unused variables
```

Comment:
TODO spotted: do you want to fix it before we merge the PR?

Reply:
Yep, this would be cool to remove the todo but I don't have the knowledge to do it. Do you thinks you can give me some explanation that I can put in this place?

Reply:
Humm, I'm not sure to understand what is blocking. The variables in the print are used later. For example, `clientID` and `clientSecret` are set in the `RegisterClientSuccessWithJustMandatoryFields` subtest, and used in the `ReadClientNoToken` subtest. If you remove the print line, what error(s) do you see?

Reply:
Ha damn sorry! Wrong todo! I was thinking about an another TODO asking for a comment why a `time.Sleep` was required in a test. 

Indeed this line can be removed and I think I forgotten some other in the previous PR. I remove this one now and I will make a PR removing the other in a few minutes